tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -70,7 +70,7 @@ const validateCardFields = async (values) => {
   }
 };
 
-describe('Target aggregates', () => {
+describe.skip('Target aggregates', () => {
```

Comment:
This e2e is already fixed in this other [PR](https://github.com/medic/cht-core/pull/9099#partial-pull-merging)